File: Basic Coding Problems/Factorial/code.java
#include<bits/stdc++.h>
using namespace std;

// Factorial of a Number

int factorial(int n){
    if(n==0 or n==1)
        return 1;
    else
        return n*factorial(n-1);
}

int main(){
    int n;
    cin>>n;

    cout<<factorial(n);
}

File: Basic Coding Problems/Palindrome/code.cpp
#include<bits/stdc++.h>
using namespace std;

// Palindrome Number

int main(){
    int n;
    cin>>n;
    int sum =0, final = n;

    while(n>0){
        int temp = n%10;
        sum = (sum*10)+temp;
        n = n/10;
    }
    if(final == sum)
        cout<<"Yes";
    else
        cout<<"No";
}
